tests/test_inplace_hooks.py

```diff
@@ -20,19 +21,24 @@ def get_hooks(pkg, backend=None, path=None):
     return Pep517HookCaller(source_dir, backend, path)
 
 
-def test_backend_path_within_tree():
-    source_dir = pjoin(SAMPLES_DIR, 'pkg1')
-    assert Pep517HookCaller(source_dir, 'dummy', ['.', 'subdir'])
-    assert Pep517HookCaller(source_dir, 'dummy', ['../pkg1', 'subdir/..'])
+@pytest.mark.parametrize('backend_path', [
+    ['.', 'subdir'],
+    ['../pkg1', 'subdir/..'],
+])
+def test_backend_path_within_tree(backend_path):
+    assert Pep517HookCaller(SOURCE_DIR, 'dummy', backend_path)
```

Comment:
```suggestion
    Pep517HookCaller(SOURCE_DIR, 'dummy', backend_path)
```

This `assert` doesn't do anything, as far as I can tell. `Pep517HookCaller` doesn't define a `__bool__` method, so it will always pass an assertion if there isn't an exception creating it.